tsconfig.json

```diff
@@ -1,13 +1,25 @@
 {
-    "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
-        "moduleResolution": "node",
-        "outDir": "./dist",
-        "rootDir": "./src",
-        "strict": true,
-        "noImplicitAny": true,
-        "esModuleInterop": true
-    },
-    "exclude": ["node_modules"]
+	"compilerOptions": {
+		"allowJs": false,
+		"allowUnreachableCode": false,
+		"allowUnusedLabels": false,
+		"esModuleInterop": true,
+		"exactOptionalPropertyTypes": true,
+		"forceConsistentCasingInFileNames": true,
+		"importsNotUsedAsValues": "error",
```

Comment:
FYI

<img width="637" alt="CleanShot 2023-03-27 at 09 33 12@2x" src="https://user-images.githubusercontent.com/6392049/228005560-4d3e2723-69f7-40a9-856c-eb34ea507de9.png">

Reply:
I noticed this error as well after attempting to upgrade to TypeScript 5; see below screenshot (from the "Features" section in the PR description) for more details.

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/26169612/228010847-d987dc40-d385-47b9-9d11-e343a7223916.png">

Reply:
Oh yes, sorry, thanks for calling this out. I think we could ignore it for now.